rasa/nlu/data_router.py

```diff
@@ -1,16 +1,12 @@
 import asyncio
```

Comment:
can we give it another try now and move the asyncio loop out of the data router `init`?

Reply:
will do it in the next PR